-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-32278/20066

ERROR: Build errors found during clang-tidy run.
```
Alignment/TrackerAlignment/plugins/CosmicRateAnalyzer.cc:58:10: error: 'DataFormats/SiPixelDetId/interface/PixelBarrelName.h' file not found [clang-diagnostic-error]
#include "DataFormats/SiPixelDetId/interface/PixelBarrelName.h"
         ^
Suppressed 1624 warnings (1623 in non-user code, 1 with check filters).
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:128: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/Alignment/TrackerAlignment/plugins/CosmicRateAnalyzer.cc
+++ b/Alignment/TrackerAlignment/plugins/CosmicRateAnalyzer.cc
@@ -153,7 +153,6 @@ private:
   std::vector<int> nh_TOB;
   std::vector<int> nh_TID;
   std::vector<int> nh_TEC;
-  
 };
 
 //
@@ -219,8 +218,8 @@ void CosmicRateAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup
 
   edm::ESHandle<MagneticField> magFieldHandle_;
   iSetup.get<IdealMagneticFieldRecord>().get(magFieldHandle_);
-  magField=-9999;
-  magField = magFieldHandle_.product()->inTesla(GlobalPoint(0,0,0)).mag();
+  magField = -9999;
+  magField = magFieldHandle_.product()->inTesla(GlobalPoint(0, 0, 0)).mag();
 
   edm::ESHandle<TrackerTopology> tTopo;
   iSetup.get<TrackerTopologyRcd>().get(tTopo);
@@ -255,7 +254,7 @@ void CosmicRateAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup
     nh_TOB.push_back(itTrack1->hitPattern().numberOfValidStripTOBHits());
     nh_TID.push_back(itTrack1->hitPattern().numberOfValidStripTIDHits());
     nh_TEC.push_back(itTrack1->hitPattern().numberOfValidStripTECHits());
-  
+
     int nhitinBPIX = 0;
     int nhitinFPIX = 0;
     int nhitinPIXEL = 0;
@@ -412,7 +411,7 @@ void CosmicRateAnalyzer::beginJob() {
   treeEvent->Branch("nHitsTID", &nh_TID);
   treeEvent->Branch("nHitsTEC", &nh_TEC);
   treeEvent->Branch("DTtime", &DTtime);
-  treeEvent->Branch("magField",&magField);
+  treeEvent->Branch("magField", &magField);
   treeRun->Branch("run_time", &run_time);
   treeRun->Branch("runnum", &runnum);
   treeRun->Branch("number_of_events", &number_of_events);

--- a/Alignment/TrackerAlignment/plugins/CosmicRateAnalyzer.cc
+++ b/Alignment/TrackerAlignment/plugins/CosmicRateAnalyzer.cc
@@ -55,8 +55,6 @@ https://twiki.cern.ch/twiki/bin/view/CMS/TkAlCosmicsRateMonitoring
 #include "DataFormats/SiStripDetId/interface/SiStripDetId.h"
 #include "DataFormats/SiPixelDetId/interface/PixelSubdetector.h"
 
-#include "DataFormats/SiPixelDetId/interface/PixelBarrelName.h"
-#include "DataFormats/SiPixelDetId/interface/PixelEndcapName.h"
 #include "MagneticField/Engine/interface/MagneticField.h"
 #include "MagneticField/Records/interface/IdealMagneticFieldRecord.h"
 #include "DataFormats/GeometryVector/interface/GlobalPoint.h"
@@ -283,7 +281,6 @@ void CosmicRateAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup
       if (PixelSubdetector::PixelBarrel == subdetId1) {
         ++nhitinBPIX;
         ++nhitinPIXEL;
-        PixelBarrelName pxbId1(detId1);
       }
       ///////////////////////////////////////////////////////////////////////////////////////////////////
       //			Hit information in PixelEndcap                                  	//

